packages/xo-vmdk-to-vhd/src/vmdk-generate.js

```diff
@@ -53,7 +53,15 @@ ddb.geometry.heads = "${geometry.heads}"
 ddb.geometry.cylinders = "${geometry.cylinders}"
 `
   const utf8Descriptor = Buffer.from(descriptor, 'utf8')
-  const descriptorSizeSectors = Math.ceil(utf8Descriptor.length / SECTOR_SIZE)
+  // virtual box add some additionnal properties to te descriptor like :
+  // ddb.uuid.image="9afa1dd0-d966-4279-a762-b7fbb0136308"
+  // ddb.uuid.modification="cd9be63c-4953-44d0-8325-45635a9ca396"
+  // ddb.uuid.parent="00000000-0000-0000-0000-000000000000"
+  // ddb.uuid.parentmodification="00000000-0000-0000-0000-000000000000"
+  // but does not ensure there is enough free room and  overwrite the data (the grain directory)
+  // broking the file
+
```

Comment:
1. typos
2. explain what's be done to work-around the issue
3. move the comment closer to the related line (and away from the unrelated one)
```suggestion

  // virtual box add some additional properties to the descriptor like:
  // ddb.uuid.image="9afa1dd0-d966-4279-a762-b7fbb0136308"
  // ddb.uuid.modification="cd9be63c-4953-44d0-8325-45635a9ca396"
  // ddb.uuid.parent="00000000-0000-0000-0000-000000000000"
  // ddb.uuid.parentmodification="00000000-0000-0000-0000-000000000000"
  //
  // but does not ensure there is enough free room and overwrite the data (the grain directory), breaking the file
  //
  // adding 10 bytes of padding seems to be enough to work-around the issue
```

Reply:
1. Try to use passive form when you can, eg "Some additional properties are added to the descriptor by Virtualbox, like:"
2. "But it doesn't check if there's enough free space and overwrite the data (the grain directory) and therefore, corrupting the file"